scripts/dev-reset.ts

```diff
@@ -44,6 +44,29 @@ function confirm(message: string): Promise<boolean> {
   )
 }
 
+async function resetRedis() {
+  const commands = [
+    'docker exec garbo_redis redis-cli flushall',
+    'podman exec -it garbo_redis bash -c "redis-cli flushall"',
```

Comment:
Good idea!

Here's a simplified command for podman
```suggestion
    'podman exec garbo_redis redis-cli flushall',
```